notebooks/SMAP/README.md

```diff
@@ -0,0 +1,25 @@
+## SMAP Python Notebooks
+
+### Summary
+
+In this set of three tutorials we demonstrate how to search for, download and plot SMAP data. Tutorial 1 demonstrates how to search for and download SMAP data using the `earthaccess` library. The second tutorial demonstrates how to read in and plot the data downloaded in Tutorial 1. And Tutorial 3 provides information on the surface quality and retrieval quality flags. 
+
+We use the [SMAP L3 Radiometer Global Daily 36 km EASE-Grid Soil Moisture, Version 8](https://nsidc.org/data/SPL3SMP/versions/8) data set as an example
+
+**NOTE** these notebooks are an updated version of the notebooks orginially published in this [repo](https://github.com/nsidc/smap_python_notebooks/tree/main). The notebooks are based on notebooks originally provided to NSIDC by Adam Purdy. Jennifer Roebuck of NSIDC applied the following updates:
```

Comment:
We may need a follow on ticket to address this repo: Do we completely remove the repo (and whatever pointers we currently have to it on NSIDC's website) and/or update to point over here.

Reply:
Probably a good idea to remove old repo.  Or remove references to it from notebooks and let original repo go stale.

Reply:
Okay I will add this as a new DUE ticket.